### PR TITLE
Add intent parameter to rtc.connect

### PR DESCRIPTION
**Pull request by feymartynov**

### CODE REVIEW

File: docs/src/api/room.md [hunk 1]
```diff
@@ -2,11 +2,11 @@
...
-backend           |     string | _required_ | Room backend, either `janus` or `none`.
-subscribers_limit |        int | _optional_ | The maximum number of simultaneously entered subscribers allowed.
+Name       | Type       | Default    | Description
+-----------| ---------- | ---------- | ----------------------------------------------------
+id         |       uuid | _required_ | The room identifier.
+audience   |     string | _required_ | The audience of the room.
+time       | [int, int] | _required_ | Opening and closing timestamps in seconds.
+created_at |        int | _required_ | Room creation timestamp in seconds.
+backend    |     string | _required_ | Room backend, either `janus` or `none`.
+reserve    |        int | _optional_ | The number of slots for agents reserved on the backend.
```
Comment (manifest) [inline docs/src/api/room.md:12]: reserve -> capacity

File: src/db/janus_backend.rs [hunk 2]
```diff
@@ -202,8 +203,8 @@ const LEAST_LOADED_SQL: &str = r#"
     LEFT JOIN room AS r2
     ON 1 = 1
     WHERE r2.id = $1
-    AND   COALESCE(jb.subscribers_limit, 2147483647) - COALESCE(jbl.taken, 0) > COALESCE(r2.reserve, 0)
-    ORDER BY COALESCE(jb.subscribers_limit, 2147483647) - COALESCE(jbl.taken, 0) DESC
+    AND   COALESCE(jb.capacity, 2147483647) - COALESCE(jbl.taken, 0) > COALESCE(r2.reserve, 0)
```
Comment (manifest) [inline src/db/janus_backend.rs:206]: `r2.reserve`?

File: docs/src/api/room.md (hunk 1 above)
Comment (feymartynov) [reply docs/src/api/room.md:12]: No. Capacity is for the backend. For rooms it's reserve.

File: src/db/janus_backend.rs (hunk 2 above)
Comment (feymartynov) [reply src/db/janus_backend.rs:206]: That's where reserves come into. The condition says that for the backend to be suitable to host the stream its free capacity must be comprise the room's required reserve.

File: docs/src/api/room.md (hunk 1 above)
Comment (manifest) [reply docs/src/api/room.md:12]: Why?

Comment (feymartynov) [reply docs/src/api/room.md:12]: There's no capacity for rooms, only reserve. Server capacity is an ability to limit the number of agents on the server. Room reserve is an ability to preliminary take a number of slots for agents. The server capacity must be big enough to accomodate this reserve.

**Pull request merged**